app/assets/stylesheets/gallery.scss

```diff
@@ -37,24 +48,28 @@ $thumbnail-active-size: $thumbnail-size + $thumbnail-margin;
   }
 
   .indicator {
-    margin: 8px 0;
-    position: unset;
-    height: $thumbnail-size + 5px;
+    bottom: 0;
+
     li {
-      border: none;
-      margin: $thumbnail-margin;
+      border: 0;
+      border-radius: $thumbnail-size / 2;
+      height: $thumbnail-size;
+      margin: 6px;
+      margin-bottom: $margin;
       vertical-align: middle;
       width: $thumbnail-size;
-      height: $thumbnail-size;
-      border-radius: $thumbnail-size / 2;
 
-      &.active, &:hover{
-        margin: $thumbnail-margin / 2;
-        width: $thumbnail-active-size;
-        height: $thumbnail-active-size;
-        border-radius: $thumbnail-active-size / 2;
-        transition: linear 0.2s;
-        transition-property: height, width, margin;
+      &.active,
```

Comment:
Nesting should be no greater than 3, but was 4

Reply:
I can't fix this one, when I put it outside of `.indicator` it asks me to merge it. IMO pronto should be set to 4, 3 is very low.